src/rust/src/rstringcache.rs

```diff
@@ -45,7 +18,6 @@ pub fn using_string_cache() -> bool {
 extendr_module! {
     mod rstringcache;
     fn enable_string_cache;
+    fn disable_string_cache;
     fn using_string_cache;
-    fn increment_string_cache_counter;
-    fn reset_string_cache;
```

Comment:
Not sure these should be removed

Reply:
makes sense to remove. 
`reset_string_cache` is now backed into `disable_string_cache`() on rust-polars side and `increment_string_cache_counter` can achived  internally via `token = .pr$RPolarsStringCacheHolder$hold()?`